### PR TITLE
Analyzer run description

### PR DESCRIPTION
**Pull request by jaegeral**
Refactoring variable names and make sure it is better explained that analyzer_run does not actually run search queries.

### CODE REVIEW

File: timesketch/lib/analyzers/sigma_tagger.py [hunk 1]
```diff
@@ -125,9 +125,9 @@ def run(self):
                                 logger.info(
                                     '[sigma] Generated query {0:s}'
                                     .format(sigma_rule))
-                                sum_of_tagged_events = self.run_sigma_rule(
+                                tagged_events_counter = self.run_sigma_rule(
                                     sigma_rule, tag_name)
-                                tags_applied[tag_name] += sum_of_tagged_events
+                                tags_applied[tag_name] += tagged_events_counter
                             except elasticsearch.TransportError \
```
Comment (berggren) [inline timesketch/lib/analyzers/sigma_tagger.py:131]: Can you take the opportunity to refactor so we don't need the "\" . Just to stay consistent with the rest of the codebase.

File: test_tools/analyzer_run.py [hunk 2]
```diff
@@ -22,6 +22,9 @@
   $ python analyzer_run.py --test_file test_file.txt \
       ../timesketch/lib/analyzers/my_analyzer.py MyAnalyzerSketchPlugin
 
+Remark: The tool ignores Elastic Search queries. It should be fed with
```
Comment (berggren) [inline test_tools/analyzer_run.py:25]:
```suggestion
Remark: The tool ignores Elasticsearch queries. It should be fed with
```

**Review by berggren — COMMENTED** [review]
Just some small nits, otherwise LGTM

File: timesketch/lib/analyzers/sigma_tagger.py (hunk 1 above)
Comment (jaegeral) [reply timesketch/lib/analyzers/sigma_tagger.py:131]: Not sure if I understand which " you mean? You mean: ```query_string='tag:"sigma*"')``` ?

Comment (berggren) [reply timesketch/lib/analyzers/sigma_tagger.py:131]: damn it, GH removed my \

Comment (berggren) [reply timesketch/lib/analyzers/sigma_tagger.py:131]: I meant, remove the / :)

**Review by berggren — CHANGES_REQUESTED** [review]
PTAL

**Review by kiddinn — APPROVED** [review]
LGTM

**Pull request merged**